bot/exts/utilities/wtf_python.py

```diff
@@ -79,14 +79,26 @@ def fuzzy_match_header(self, query: str) -> Optional[str]:
         return match if certainty > MINIMUM_CERTAINTY else None
 
     @commands.command(aliases=("wtf", "WTF"))
-    async def wtf_python(self, ctx: commands.Context, *, query: str) -> None:
+    async def wtf_python(self, ctx: commands.Context, *, query: Optional[str] = None) -> None:
         """
         Search WTF Python repository.
 
         Gets the link of the fuzzy matched query from https://github.com/satwikkansal/wtfpython.
         Usage:
             --> .wtf wild imports
         """
+        if query is None:
+            no_query_embed = Embed(
+                title="WTF Python?!",
+                colour=constants.Colours.dark_green,
+                description="A repository filled with suprising snippets that can make you say WTF?!\n"
```

Comment:
Currently it looks like this:
![image](https://user-images.githubusercontent.com/50042066/140594994-f3ceab4b-eb81-433b-810a-3796ec927bf4.png)
```suggestion
                description="A repository filled with suprising snippets that can make you say WTF?!\n\n"
```
I think double newline looks a bit better. Perhaps we could switch to the discord link button (like `.challenge`) when we start using it more often in other places, but this is good for now.